clients/ui/frontend/src/app/pages/settings/roleBinding/RoleBindingPermissionsTable.tsx

```diff
@@ -30,6 +30,10 @@ type RoleBindingPermissionsTableProps = {
   typeAhead?: string[];
   createRoleBinding: (roleBinding: RoleBindingKind) => Promise<RoleBindingKind>;
   deleteRoleBinding: (name: string, namespace: string) => Promise<K8sStatus>;
+  tryPatchRoleBinding: (
+    oldRBObject: RoleBindingKind,
+    newRBObject: RoleBindingKind,
+  ) => Promise<boolean>;
```

Comment:
same here, we should not pass tryPatchRoleBinding as prop here, as we directly import in in RoleBindingPermissionsTable component. these changes should be reverted.